test/unit/express.test.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+// load default gitvariables for testing
```

Comment:
Do you mean `git variables`

Reply:
I was probably typing a git command hahah